packages/get-platform/src/getPlatform.ts

```diff
@@ -206,5 +239,36 @@ export async function getPlatform(): Promise<Platform> {
   }
 
   // use the debian build with OpenSSL 1.1 as a last resort
+  // TODO: perhaps we should default to 'debian-openssl-3.0.x'
```

Comment:
btw, if we still had the real statically linked musl build, like before introduction of Node-API library, as opposed to just calling alpine-specific build "musl", we could use it as the default — it would work on absolutely any linux distribution (at the expense of larger binaries)

Reply:
we should look into bringing back statically linked musl later and see if we could make it work with Node-API, otherwise we can't support distroless images